src/app/places/forms/update.tsx

```diff
@@ -0,0 +1,38 @@
+import { usePlacesContext } from '@/context/PlacesContext';
+import { Box, Button, Stack, TextField } from '@mui/material';
+import { Place } from '@prisma/client';
+import React from 'react';
+import { MdEdit } from 'react-icons/md';
+
+interface Props {
+  place: Place;
+}
+
+export const PlaceUpdateForm = ({ place }: Props) => {
+  const { closeDialog, updatePlace, isLoading } = usePlacesContext();
+
+  const onSubmit = async (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+    const data = new FormData(event.currentTarget);
+    await updatePlace({
+      id: data.get('id'),
+      title: data.get('title'),
+    } as Place);
+    closeDialog();
+  };
+
+  return (
+    <Box component='form' onSubmit={onSubmit} noValidate>
+      <input type='hidden' name='id' value={place.id} />
```

Comment:
Is there a reason to use hidden type field? Can we just send it directly to updatePlaace